framework/src/test/java/org/tron/core/actuator/utils/ProposalUtilTest.java

```diff
@@ -0,0 +1,302 @@
+      Assert.assertTrue(false);
+    } catch (ContractValidateException e) {
+      Assert.assertTrue(e instanceof ContractValidateException);
+    }
+
+
+    try {
+      actuatorUtil.validator(dynamicPropertiesStore, forkUtils,
+          ProposalType.CREATE_NEW_ACCOUNT_FEE_IN_SYSTEM_CONTRACT.getCode(), invalidValue);
+      Assert.assertTrue(false);
+    } catch (ContractValidateException e) {
+      Assert.assertTrue(e instanceof ContractValidateException);
+    }
+
+    try {
+      actuatorUtil.validator(dynamicPropertiesStore, forkUtils,
+          ProposalType.CREATE_NEW_ACCOUNT_FEE_IN_SYSTEM_CONTRACT.getCode(), LONG_VALUE + 1);
+      Assert.assertTrue(false);
+    } catch (ContractValidateException e) {
+      Assert.assertTrue(e instanceof ContractValidateException);
+    }
+
+    try {
+      actuatorUtil.validator(dynamicPropertiesStore, forkUtils,
+          ProposalType.CREATE_NEW_ACCOUNT_BANDWIDTH_RATE.getCode(), invalidValue);
+      Assert.assertTrue(false);
+    } catch (ContractValidateException e) {
+      Assert.assertTrue(e instanceof ContractValidateException);
+    }
+
+    try {
+      actuatorUtil.validator(dynamicPropertiesStore, forkUtils,
+          ProposalType.CREATE_NEW_ACCOUNT_BANDWIDTH_RATE.getCode(), LONG_VALUE + 1);
+      Assert.assertTrue(false);
+    } catch (ContractValidateException e) {
+      Assert.assertTrue(e instanceof ContractValidateException);
+    }
+
+
+    long value = 32;
+    try {
+      actuatorUtil.validator(dynamicPropertiesStore, forkUtils,
+          ProposalType.MAINTENANCE_TIME_INTERVAL.getCode(), 3 * 27 * 1000 - 1);
+      Assert.assertTrue(false);
+    } catch (ContractValidateException e) {
+      Assert.assertTrue(e instanceof ContractValidateException);
+      Assert.assertEquals(
+          "Bad chain parameter value, valid range is [3 * 27 * 1000,24 * 3600 * 1000]",
+          e.getMessage());
+    }
+
+    try {
+      actuatorUtil.validator(dynamicPropertiesStore, forkUtils,
+          ProposalType.MAINTENANCE_TIME_INTERVAL.getCode(), 24 * 3600 * 1000 + 1);
+      Assert.assertTrue(false);
+    } catch (ContractValidateException e) {
+      Assert.assertTrue(e instanceof ContractValidateException);
+      Assert.assertEquals(
+          "Bad chain parameter value, valid range is [3 * 27 * 1000,24 * 3600 * 1000]",
+          e.getMessage());
+    }
+
+
+    try {
+      actuatorUtil.validator(dynamicPropertiesStore, forkUtils,
+          ProposalType.ALLOW_CREATION_OF_CONTRACTS.getCode(), 2);
+      Assert.assertTrue(false);
+    } catch (ContractValidateException e) {
+      Assert.assertTrue(e instanceof ContractValidateException);
+      Assert.assertEquals(
+          "This value[ALLOW_CREATION_OF_CONTRACTS] is only allowed to be 1",
+          e.getMessage());
+    }
+
+    dynamicPropertiesStore = dbManager.getDynamicPropertiesStore();
+    try {
+      actuatorUtil.validator(dynamicPropertiesStore, forkUtils,
```

Comment:
-1 should be validated for completeness.

Reply:
correct. Will fix, please take a look at next commit